iam.main.tf

```diff
@@ -193,6 +196,8 @@ resource "aws_iam_role_policy" "ConsoleTaskPolicy" {
           "arn:aws:cloudwatch:*:*:alarm:*",
           "arn:aws:ec2:*::image/*",
           "arn:aws:ec2:*::volume/*",
```

Comment:
At some point, we need to update these resources [subbing in the correct AWS partition](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/partition)...